app/templates/components/nav.twig

```diff
@@ -3,44 +3,23 @@
 
         <!-- <img src="/web-assets/images/BDPA-Flights-Black.jpeg" class="rounded" width="44" height="44" alt="" loading="lazy"> -->
 
-        <a href="<?=$_ENV['BASE_URL']?>" class="navbar-brand mb-0 h1 ml-3">Ghostmeme</a> <!-- Make name Airlanta -->
+        <a href="#" class="navbar-brand mb-0 h1 ml-3">Ghostmeme</a> <!-- Make name Airlanta -->
         <div class="col col-md w-100 justify-content-center">
-            <form method='get' action="<?=$_ENV['BASE_URL']?>">
+            <form method='get' action="/search.php">
```

Comment:
`search.php` doesn't exist, this is just for the twig example- another "fixme"

Reply:
Right, I did the searching in index.php. I think I'll make a search.php to handle the searching. I don't know why I did it in index having a separate file makes more sense.